### ISSUE TITLE
README doesn't document all inputs for github action ansible/ansible-lint

### ISSUE THREAD

**Issue by dobbymoodge** [ansible/ansible-lint#4692]
##### Summary

README.md only documents 4 of the `ansible/ansible-lint` inputs for the GitHub action defined in `action.yml`:  `args`, `setup_python`, `working_directory`, `requirements_file`. The other two actions, `expected_return_code` and `gh_action_ref` are not documented

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

GitHub Actions

##### STEPS TO REPRODUCE

Read the `README.md` looking for an explanation for why

```yaml
      - name: Ansible-lint
        uses: ansible/ansible-lint@v25.6.1    # This tells GH Actions to pull down the pinned version
```

installs the wrong version of `ansible-lint` in the action environment, leading to this in the output:

```
ansible-lint 25.6.2.dev25 using ansible-core:2.19.0 ansible-compat:25.6.0 ruamel-yaml:0.18.14 ruamel-yaml-clib:0.2.12
```
...and for the linter to flag up errors that are inconsistent with those identified by the version of `ansible-lint` you had asked for.

##### Desired Behavior

Complete and correct documentation.

##### Actual Behavior

The documentation should include details about all inputs to the GitHub action, and should also explain why [the version specifier in the `uses:` clause is ignored](https://github.com/ansible/ansible-lint/issues/3830).

**Comment by alisonlhart** [comment]
#4701 will close this. Note: I did not add `expected_return_code` to the documentation because this is an input only intended for internal testing use. I have added a note about this to the documentation on the input itself.
